client/src/GlobalStyles.js

```diff
@@ -109,6 +109,103 @@ const GlobalStyles = () => {
           padding: 0 1em 0 1em;
         }
 
+        /* MAIN PAGE  */
+
+        .main--main {
+          flex: 1;
+          padding: 0 1em 0 1em;
+        }
+
+        /* MAIN SUBSCRIBE SECTION  */
+
+        .subscribe__section {
```

Comment:
You define a lot of CSS classes in addition to your styled components. I wouldn't mix it and focus on styled components or CSS, not both.

Reply:
Sure! I will move almost everything to styled components! Thanks for the advice!